src/main/java/uk/gov/hmcts/reform/translate/helper/DictionaryUtils.java

```diff
@@ -0,0 +1,29 @@
+package uk.gov.hmcts.reform.translate.helper;
+
+import org.apache.commons.lang3.StringUtils;
+import uk.gov.hmcts.reform.translate.model.Dictionary;
+
+import java.util.Map;
+import java.util.function.Predicate;
+
+
+public final class DictionaryUtils {
+
+    private static final Predicate<String> isTranslationNotNull = translation -> !StringUtils.isEmpty(translation);
+
+    public static boolean isTranslationBodyEmpty(final Dictionary dictionaryRequest) {
+        return dictionaryRequest.getTranslations() == null || dictionaryRequest.getTranslations().isEmpty();
+    }
+
+    public static boolean hasAnyTranslations(final Dictionary dictionaryRequest) {
+        return dictionaryRequest.getTranslations().values().stream().anyMatch(isTranslationNotNull);
+    }
+
+    public static boolean hasTranslationPhrase(final Map.Entry<String, String> currentPhrase) {
+        return isTranslationNotNull.test(currentPhrase.getValue());
```

Comment:
Do we need the predicate if it's not being reused elsewhere

Could make method bit easier to read

`return !StringUtils.isEmpty(currentPhrase.getValue)`

Reply:
Yes :) It is used in other places.